server/handlers/get_account.go

```diff
@@ -27,11 +27,27 @@ func GetAccount(app *app.App) http.HandlerFunc {
 			panic(err)
 		}
 
+		var formattedLastLogin string
+		if account.LastLoginAt == nil {
+			formattedLastLogin = ""
+		} else {
+			formattedLastLogin = account.LastLoginAt.Format("2006-01-02T15:04:05Z07:00")
+		}
+
+		var formattedPasswordChangedAt string
+		if account.PasswordChangedAt.IsZero() {
+			formattedPasswordChangedAt = ""
+		} else {
+			formattedPasswordChangedAt = account.PasswordChangedAt.Format("2006-01-02T15:04:05Z07:00")
```

Comment:
i went looking for a standard iso 8601 formatter and discovered:

```suggestion
			formattedPasswordChangedAt = account.PasswordChangedAt.Format(time.RFC3339)
```

Reply:
👍